src/CGB_Infrared.md

```diff
@@ -0,0 +1,49 @@
+# CGB Infrared
+
+The GBC comes with an infrared port on the very top of the handheld,
+which can be used to send and receive IR signals via its two separate
+diodes. This functionality is not limited to other GBCs - it can be
+used to communicate with external devices as well.
+
+Note that the GBA does not include an infrared port and, as such, this
+method of communication is unavailable in its CGB compatibility mode.
+
+## FF56 - RP - Infrared Communications Port
+
+This register allows data to be transmitted and received through the
+infrared port.
+
+Bit | Name             | Usage notes                     | R/W?
```

Comment:
The format of these is being discussed in #318, feel free to weigh in there if this is the style you prefer (maybe not, since it looks like you may have sliced this section from the original docs verbatim).

In the meantime, I'd like this be kept consistent with the style used throughout the rest of the document ([such as here](https://gbdev.io/pandocs/STAT.html#ff41---stat-lcd-status-rw)).

Reply:
I've based it on the LCDC page, as I assumed this is the current "desired" style. The original docs's format differs.

Reply:
Yeah, we know the style is not used consistently, hence that issue. :/

Reply:
(By the way, I'm picking this one specifically as I believe it's the best compromise until the new syntax is implemented.)